pkgs/development/python-modules/regex/default.nix

```diff
@@ -22,10 +21,12 @@ buildPythonPackage rec {
   # No tests in archive
   doCheck = false;
```

Comment:
Can we fetch from bitbucket and get the tests running?

Reply:
Test are enabled now.